guides/README.md

```diff
@@ -67,6 +67,22 @@ It's also possible to check links; the following command will check all links ex
 
 	make linkchecker
 
+## Building Locally Using a Container
+
+You can build Foreman documentation locally using a container image.
+This requires the cloned git repository plus an application such as Podman or Docker to build and run container images.
+
+1. Build container image:
+
+       docker build --no-cache --progress=plain --file Containerfile --tag foreman_documentation:1.0 .
+
+2. Build Foreman documentation.
+   Run this command in your `foreman-documentation` repository:
+
+       rm -rf guides/build && docker run --rm -v $(pwd):/foreman-documentation foreman_documentation:1.0 make html
+
+   Add the `-Z` flag on SELinux enabled systems.
```

Comment:
Is this related?

```
[lzap@nuc foreman-documentation]$ docker run --rm -v $(pwd):/foreman-documentation foreman_documentation:1.0 make html
make: stat: Makefile: Permission denied
make: *** No rule to make target 'html'.  Stop.
```

Reply:
I can only do this with sudo.

Reply:
I cannot reproduce this, but I've added a note to run this as root on systems using SELinux. I am unsure if this makes it user friendly.

Reply:
Ok I know what is going on. Users do not map correctly, my ID is 1001 (yeah I am special) and this does not map to root:

```
$ docker run --rm -v $(pwd):/foreman-documentation -it foreman_documentation:1.0 /bin/bash
[root@21ecf14c6fda /]# ls -la /
total 16
dr-xr-xr-x.   1 root root  42 Aug 12 16:25 .
dr-xr-xr-x.   1 root root  42 Aug 12 16:25 ..
drwxrwxr-x.   1 1001 1001 224 Aug 12 14:25 foreman-documentation
...
```

Within the container environment, the effective user (which is UID=0) cannot do anything with the contents. I am not container expert, not sure what to do here.

Reply:
We have a similar script in pulpcore-packaging and here's how we deal with SELinux:
https://github.com/theforeman/pulpcore-packaging/blob/2b69c4af86d45833d3530ba9e78db301fc25abe5/automation/update_packaging.sh#L16-L22